docs/tutorials/07-batch-infer.ipynb

```diff
@@ -7,7 +7,7 @@
    "source": [
```

Comment:
<p>You can zoom into one of the model endpoints to get an <a href="../monitoring/model-monitoring-deployment.html#model-endpoint-summary-list" target="_blank">overview</a> about the selected endpoint, including the calculated statistical drift metrics:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/4273/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>